frontends/web/src/components/password.tsx

```diff
@@ -302,9 +340,10 @@ class PasswordRepeatInputClass extends Component {
   }
 }
 
-export const PasswordRepeatInput = withTranslation(null, { withRef: true })(PasswordRepeatInputClass);
+const HOCRepeat = translate(undefined, { withRef: true })(PasswordRepeatInputClass);
+export { HOCRepeat as PasswordRepeatInput };
 
-const MatchesPattern = ({ regex, value = '', text }) => {
+const MatchesPattern = ({ regex, value = '', text }: { regex: RegExp | undefined, value: string, text: string | undefined }) => {
```

Comment:
please change to `type MatchesPatternProps = {`

and use MatchesPatternProps here.